client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import debugModule from 'debug';
+import {
+	identity,
+	omit,
+} from 'lodash';
+import { localize } from 'i18n-calypso';
+const debug = debugModule( 'calypso:my-sites:people:team-list' );
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import PeopleListItem from 'my-sites/people/people-list-item';
+import UsersActions from 'lib/users/actions';
+import InfiniteList from 'components/infinite-list';
+import deterministicStringify from 'lib/deterministic-stringify';
+import NoResults from 'my-sites/no-results';
+import PeopleListSectionHeader from 'my-sites/people/people-list-section-header';
+import { recordGoogleEvent } from 'state/analytics/actions';
+
+/**
+ * Module Variables
+ */
+export class Team extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	isLastPage = () => this.props.totalUsers <= this.props.users.length + this.props.excludedUsers.length;
```

Comment:
this code here is vulnerable to throwing `TypeErrors` when `this.props.users` or `this.props.excludedUsers` is undefined. we could use safe getters instead

```js
isLastPage = () => {
	const {
		excludedUsers = [],
		users = [],
		totalUsers,
	} = this.props;

	return totalUsers < ( users.length + excludedUsers.length );
}
```

alternatively we could provide default props, though the safety issue does technically exist (could be non-arrays) but this method or the use of `get()` could eliminate most of the risk